hypervisor/include/common/hypercall.h

```diff
@@ -51,6 +51,20 @@ int32_t hcall_sos_offline_cpu(struct acrn_vm *vm, uint64_t lapicid);
  */
 int32_t hcall_get_api_version(struct acrn_vm *vm, uint64_t param);
 
+
+/**
+ * @brief Get basic platform information
+ *
+ * The function return basic platform configurations information.
```

Comment:
Should we make this paragraph the exact copy of the one in the `.c` file (or vice-versa)